edumap/config/initializers/contact_us.rb

```diff
@@ -0,0 +1,34 @@
+# Use this hook to configure contact mailer.
+ContactUs.setup do |config|
+
+  # ==> Mailer Configuration
+
+  # Configure the e-mail address which email notifications should be sent from.  If emails must be sent from a verified email address you may set it here.
+  # Example:
+  # config.mailer_from = "contact@please-change-me.com"
+  config.mailer_from = nil
+
+  # Configure the e-mail address which should receive the contact form email notifications.
+  config.mailer_to = "edumapcs@gmail.com" #change this to contact email
```

Comment:
Might be a good idea, but it's not something I've done before.  I don't think I'll have any extra time this week to research it myself, but we can look into it on Tuesday.